### PR TITLE
Add volume for Docker container & Swagger fixes

### LINKED ISSUE #5: Add support for saving documents to backend

### LINKED ISSUE #6: Add support for saving project data to backend

### PR DESCRIPTION
**Pull request by ghost**
resolves #5 
resolves #6

### CODE REVIEW

**Review by katzper-michno — APPROVED** [review]

File: swagger.yaml [hunk 1]
```diff
@@ -100,6 +80,18 @@ paths:
...
+        - auth
+      summary: Delete session
+      description: Deletes the current session
+      security:
+        - session_id: []
+      responses:
+        204:
+          description: Successfully deleted session
+        400:
+          description: Malformed request
```
Comment (AHGIJMKLKKZNPJKQR) [inline swagger.yaml:94]: 401 if no cookie in request and 422 if unrecognized RSESSID value format

**Review by AHGIJMKLKKZNPJKQR — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 1 above)
Comment (ghost) [reply swagger.yaml:94]: To be honest I copied it from resource management swagger (:

Comment (AHGIJMKLKKZNPJKQR) [reply swagger.yaml:94]: 400 can still occur I think if someone sends something really messed up. Like by malformed request I understand something that can barely be recognized as an HTTP request. The 422 is more for validating if the session ID length (and I think charset) is correct.

Comment (ghost) [reply swagger.yaml:94]: Gotcha

**Review by AHGIJMKLKKZNPJKQR — APPROVED** [review]

**Pull request merged**